src/components/form/fields/radio-group/RadioGroup.tsx

```diff
@@ -0,0 +1,39 @@
+import { ReqoreRadioGroup } from '@qoretechnologies/reqore';
+import { IReqoreRadioGroupProps } from '@qoretechnologies/reqore/dist/components/RadioGroup';
+
+export interface IRadioGroupFormFieldProps
+  extends Omit<IReqoreRadioGroupProps, 'onChange' | 'onSelectClick' | 'selected'> {
+  value: string;
+  onChange(value: string): void;
+  disabled?: boolean;
+}
+
+export const FormRadioGroupField = ({
+  items,
+  disabled,
+  onChange,
+  value,
+  ...rest
+}: IRadioGroupFormFieldProps) => {
+  return (
+    <ReqoreRadioGroup
+      items={items.map((item) => ({
```

Comment:
maybe move this into a `useMemo` var so the object is not recreated on every render?